content/twim-guide.md

```diff
@@ -27,28 +27,28 @@ Matrix room.
 After joining you should start your post with a mention of the user "TWIM".
 Easiest to do this is to use your client's mention completion feature.
 After that mention everything contained in the message will be part of the entry.
-If you want to post more than one project please split each into individual entries.
+If you want to post more than one project please submit individual entries, one for each project.
 
 Images and other media are supported as well. To add them you should react with
-`⭕` on your media file. Please make sure to do this after your entry so it gets added
-to the correct entry and not on another project of yours.
+`⭕` on your media file. Please make sure to do this right after submitting your text so it gets added
+to the correct entry and not to another project of yours.
 
 ## How to make it a good entry
 
 Making a good entry can have some challenges. Below are some recommendations
-to help the authors to lower their workload when publishing the post.
+to help the editors to lower their workload when publishing the post.
 
 ### Markdown
 
-The bot expected markdown as part of the plaintext event body. Please use a client which
-does send the plaintext as markdown. While we currently accept other formats as well
+The bot expects markdown as part of the plaintext event body. Please use a client which
+does include the markdown in the plaintext. While we currently accept other formats as well
 it will add manual work for the post by a human.
 
 ### Do not treat it as a Changelog
 
 Copying a Changelog is easy but not what TWIM is about. Ideally we prefer posts
-which are a few paragraphs long outlining features which were added or changed
-over a bulletpoint list. This makes it more readable to humans. You may still of course
+which are a few paragraphs long outlining some highlights
+over a long detailed list. This makes it more readable to humans. You may of course still
```

Comment:
Hm isnt this the opposite of the original? A bulletpoint list for me is a list of not detailed things. You essentially just have headers in it.

Reply:
- I think having bullet points as a semantic element is OK, e.g. see element X updates. I would actually prefer it to a paragraph where I only know after reading how many topics were covered.
- I think posts, whether they are written as text or 1 sentence per bullet point, should focus on highlights and not every single detail (such as commit messages). That level of detail should be included by linking to release notes/changelog if desired.